notebooks/Periodicity-Detection/Periodicity_Detection.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>I think it would be good to add a section about "Applications" and state how this could be used in security-related fields. Also it might be nice to provide some end-to-end timings just to see how fast it is on GPUs. You can do this without showing the data but using actual data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>